src/diffusers/models/attention.py

```diff
@@ -439,7 +435,7 @@ def _set_attention_slice(self, slice_size):
         self.attn1._slice_size = slice_size
         self.attn2._slice_size = slice_size
 
-    def _set_use_memory_efficient_attention_xformers(self, use_memory_efficient_attention_xformers: bool):
+    def set_use_memory_efficient_attention_xformers(self, use_memory_efficient_attention_xformers: bool):
```

Comment:
alled from the outside so can be public ? Plus conveys the idea that it's a capability being exposed